config/buildExample.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import { parse } from "@babel/parser";
```

Comment:
no need to add `@babel/parser` to dependencies, `@babel/core` already has parser inside :)